web/libs/editor/src/core/settings/keymap.json

```diff
@@ -45,6 +45,16 @@
     "mac": "alt+enter",
     "description": "Skip task"
   },
+  "annotation:next": {
```

Comment:
I think we have a problem with the naming. We're not moving to the next and previous annotations; we're moving to the next and previous tasks. We need to fix the naming